Author: Philip Rinn <rinni@inventati.org>
Description: Disable aarch64 feature as it uses unstable features of rustc
Last-update: 2022-06-30
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,7 +50,6 @@
 optional = true
 
 [features]
-aarch64 = []
 alloc = []
 default = ["alloc"]
 derive = ["zeroize_derive"]
